docs/topics/basket.rst

```diff
@@ -0,0 +1,149 @@
+Basket Synchronisation
+======================
+
+This documents what data we synchronize with `Basket <https://basket.readthedocs.io/>`_  and how.
+
+Triggers
+--------
+
+Every time a field that we're meant to synchronize on an object on changes, a full sync of the
+object is triggered.
+
+A consequence of this is, since the relation between an addon and an user is part of the add-on
+object, when a new user is added as an author, or an existing author is removed from an add-on,
+that triggers a full sync of the add-on, including user account information for all its authors.
+
+Objects
+-------
+
+We're synchronizing 2 types of objects:
+
+    - User Accounts
+    - Add-ons
+
+
+User Accounts
+~~~~~~~~~~~~~
+
+.. note::
+     Newsletter opt-in information is not stored by AMO, and therefore not synchronized with the
+     rest. It's sent to basket separately directly whenever it changes, through basket's
+     `Newsletter API <https://basket.readthedocs.io/newsletter_api.html>`_ ``subscribe`` and
+     ``unsubscribe`` endpoints.
+
+.. http:post:: https://basket.mozilla.org/amo-sync/userprofile/
+
+    :<json int id: The numeric user id.
+    :>json boolean deleted: Is the account deleted.
+    :<json string|null display_name: The name chosen by the user.
```

Comment:
mix of inputs and outputs here - or is `deleted` an input too?

Reply:
Everything is an input, typo on my part.

Reply:
49ee7c029c2132559456a8be2af7240f49afa603